src/App.tsx

```diff
@@ -1,7 +1,36 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.showLetter);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.showLetter);
+  }
+
+  showLetter = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is ${event.key}` });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.pressedKey === ''
```

Comment:
```suggestion
        {this.state.pressedKey
```
you may check it in this way - an empty string will give you false

FYI